docs/modules/ROOT/pages/how-tos/vsphere/install.adoc

```diff
@@ -0,0 +1,189 @@
+----
+export VSPHERE_CLUSTER=<cluster name>
+export VSPHERE_DATACENTER=<datacenter name>
+export VSPHERE_DATASTORE=<datastore name>
+export VSPHERE_FOLDER=<folder name>
+export VSPHERE_NETWORK=<network name>
+----
+
+.Cluster machine network
+[source,bash]
+----
+export MACHINE_NETWORK_CIDR=<machine network cidr>
+export API_VIP=<api vip>
+export INGRESS_VIP=<ingress vip>
+----
+
+include::partial$install/vshn-input.adoc[]
+
+[#_set_vault_secrets]
+=== Set secrets in Vault
+
+include::partial$connect-to-vault.adoc[]
+
+.Store various secrets in Vault
+[source,bash]
+----
+# Store vSphere credentials
+vault kv put  clusters/kv/${TENANT_ID}/${CLUSTER_ID}/vsphere/credentials \
+  username=${VSPHERE_USERNAME} \
+  password=${VSPHERE_PASSWORD}
+
+# Generate an HTTP secret for the registry
+vault kv put clusters/kv/${TENANT_ID}/${CLUSTER_ID}/registry \
+  httpSecret=$(LC_ALL=C tr -cd "A-Za-z0-9" </dev/urandom | head -c 128)
+
+# Set the LDAP password
+vault kv put clusters/kv/${TENANT_ID}/${CLUSTER_ID}/vshn-ldap \
+  bindPassword=${LDAP_PASSWORD}
+
+# Generate a master password for K8up backups
+vault kv put clusters/kv/${TENANT_ID}/${CLUSTER_ID}/global-backup \
+  password=$(LC_ALL=C tr -cd "A-Za-z0-9" </dev/urandom | head -c 32)
+
+# Generate a password for the cluster object backups
+vault kv put clusters/kv/${TENANT_ID}/${CLUSTER_ID}/cluster-backup \
+  password=$(LC_ALL=C tr -cd "A-Za-z0-9" </dev/urandom | head -c 32)
+
+# Copy the VSHN acme-dns registration password
+vault kv get -format=json "clusters/kv/template/cert-manager" | jq '.data.data' \
+  | vault kv put -cas=0 "clusters/kv/${TENANT_ID}/${CLUSTER_ID}/cert-manager" -
+
+----
+
+include::partial$install/prepare-commodore.adoc[]
+
+[#_configure_installer]
+=== Configure the OpenShift Installer
+
+include::partial$install/configure-installer.adoc[]
+
+[#_prepare_installer]
+=== Prepare the OpenShift Installer
+
+include::partial$install/run-installer.adoc[]
+
+=== Update Project Syn cluster config
+
+include::partial$install/prepare-syn-config.adoc[]
+
+=== Provision the cluster
+
+[NOTE]
+====
+The steps in this section must be run on a host which can reach the vSphere API.
+If you can't reach the vSphere API directly, you can setup a SOCKS5 proxy with the following commands:
+
+[source,bash]
+----
+export JUMPHOST_FQDN=<jumphost fqdn or alias from your SSH config> <1>
+ssh -D 12000 -q -f -N ${JUMPHOST_FQDN} <2>
+export VCENTER_API_URL="https://$(ssh ${JUMPHOST_FQDN} dig +short ${VCENTER_HOSTNAME})" <2>
+export https_proxy=socks5://localhost:12000 <3>
+----
+<1> The FQDN or SSH alias of the host which can reach the vSphere API
+<2> This command expects that your SSH config is setup so that `ssh ${JUMPHOST_FQDN}` works without further configuration
+<3> The `openshift-install` tool respects the `https_proxy` environment variable
+====
+
+. Trust the vSphere CA certificate
++
+.Ubuntu 22.04
+[source,bash]
+----
+curl -kLo vsphere-ca.zip ${VCENTER_API_URL}/certs/download.zip <1>
```

Comment:
Footnote is missing here afaics

Reply:
I figured out how to tell curl to resolve DNS names over a SOCKS5 proxy, so I've updated this section slightly.